barretenberg_wrapper/build.rs

```diff
@@ -215,7 +215,10 @@ fn link_lib_omp(toolchain: &'static str) {
             let llvm_dir = find_llvm_linux_path();
             println!("cargo:rustc-link-search={}/lib", llvm_dir)
         }
-        INTEL_APPLE => println!("cargo:rustc-link-search=/usr/local/lib"),
+        INTEL_APPLE => {
+            let brew_prefix = find_brew_prefix();
+            println!("cargo:rustc-link-search={}/lib", brew_prefix)
```

Comment:
@vezenovm Do we want this to be 
```suggestion
            println!("cargo:rustc-link-search={}/opt/libomp/lib", brew_prefix)
```
just in case brew decides it doesn't want to create the symlink in `{prefix}/lib`?

Reply:
Ah I guess this was the original reason you opened the PR. I am able to build with that suggested change so I say we do it. It looks like my brew is symlinking in `{prefix}/lib` so I just didn't run into the same error you did.

Reply:
I just pushed the change